### PR TITLE
AArch64: Implement static PICs for virtual and interface call

### PR DESCRIPTION
**Pull request by Akira1Saitoh**
Implement static PICs for virtual and interface call on aarch64.
This implementation is largely based on ppc's one.

Signed-off-by: Akira Saitoh <saiaki@jp.ibm.com>

### CODE REVIEW

File: runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp [hunk 1]
```diff
@@ -1601,6 +1770,95 @@ void J9::ARM64::PrivateLinkage::buildVirtualDispatch(TR::Node *callNode,
...
+            TR_ASSERT_FATAL(!OOLLabelInstr->getLiveLocals() && !OOLLabelInstr->getLiveMonitors(), "Expecting first OOL instruction to not have live locals/monitors info");
+            OOLLabelInstr->setLiveLocals(gcPoint->getLiveLocals());
+            OOLLabelInstr->setLiveMonitors(gcPoint->getLiveMonitors());
+
+            TR::LabelSymbol *doneOOLLabel = generateLabelSymbol(cg());
+
+            if (methodSymbol->isInterface())
+               {
+               TR::LabelSymbol              *ifcSnippetLabel = generateLabelSymbol(cg());
+               TR::ARM64InterfaceCallSnippet *ifcSnippet =     new (trHeapMemory()) TR::ARM64InterfaceCallSnippet(cg(), callNode, ifcSnippetLabel, argSize, doneOOLLabel, (uint8_t *)thunk);
```
Comment (knn-k) [inline runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp:1824]: Please align the column position of the variables.

Comment (Akira1Saitoh) [reply runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp]: I fixed it.

**Comment by knn-k** [comment]
jenkins test sanity alinux64 jdk11

**Comment by knn-k** [comment]
jenkins test extended.functional alinux64 jdk11

**Comment by knn-k** [comment]
jenkins test extended.functional alinux64 jdk11

**Comment by knn-k** [comment]
The failures with  jit_jitt_array_compress_0/1 are a known problem #12947.

**Review by knn-k — APPROVED** [review]

**Pull request merged**